modules/programs/alacritty.nix

```diff
@@ -5,6 +5,270 @@ with lib;
 let
   cfg = config.programs.alacritty;
   yamlFormat = pkgs.formats.yaml { };
+  CSIuKeyBindings = [ # http://www.leonerd.org.uk/hacks/fixterms/
+    # Avoid to override alacritty default key-bindings
+    #
+    # Control + B ⇒ SearchBackward
+    # Control + C ⇒ Copy
+    # Control + F ⇒ SearchForward
+    # Control + V ⇒ Paste
+    # Control + 0 ⇒ ResetFontSize
+
+    {
+      key = "H";
+      mods = "Control";
+      chars = "\\x1b[104;5u";
+    }
+    {
+      key = "I";
+      mods = "Control";
+      chars = "\\x1b[105;5u";
+    }
+    {
+      key = "J";
+      mods = "Control";
+      chars = "\\x1b[106;5u";
+    }
+    {
+      key = "M";
+      mods = "Control";
+      chars = "\\x1b[109;5u";
+    }
+
+    {
+      key = "Return";
+      mods = "Control";
+      chars = "\\x1b[13;5u";
+    }
+    {
+      key = "Slash";
+      mods = "Control";
+      chars = "\\x1b[47;5u";
+    }
+    {
+      key = "51";
+      mods = "Control|Shift";
+      chars = "\\x1b[60;5u";
+    } # Less
+    {
+      key = "52";
+      mods = "Control|Shift";
+      chars = "\\x1b[62;5u";
+    } # Greater
+
+    {
+      key = "A";
+      mods = "Control|Shift";
+      chars = "\\x1b[65;5u";
+    }
+    # { key = "B"; mods = "Control|Shift"; chars = "\\x1b[66;5u"; }
+    # { key = "C"; mods = "Control|Shift"; chars = "\\x1b[67;5u"; }
```

Comment:
Why are these commented?
(Same goes for the other commented keys)

Reply:
Those are alactritty native bindings, so &mdash; for alacritty &mdash; those must be exempted. For another shell things will look different.